app/serializers/journey_serializer.rb

```diff
@@ -11,6 +11,10 @@ class JourneySerializer
   has_one :from_location, serializer: LocationSerializer
   has_one :to_location, serializer: LocationSerializer
 
+  has_many :events, serializer: GenericEventSerializer do |object|
+    object.generic_events.applied_order
```

Comment:
will using `applied_order` scope cause n+1 queries on the eventables?

Reply:
Possibly. I'll talk to Martyn to avoid this situation when indexing moves.

Reply:
Looking at the output from bullet it doesn't complain about n+1s here 

**when showing an individual move:**

```
GET /api/moves/d17c348f-ae1c-4385-9899-0b711fe17e19?include=events
AVOID eager loading detected
  Move => [:from_location, :to_location, :profile]
  Remove from your query: .includes([:from_location, :to_location, :profile])
Call stack

2020-10-21 16:38:33[WARN] user: william
GET /api/moves/d17c348f-ae1c-4385-9899-0b711fe17e19?include=events
AVOID eager loading detected
  Profile => [:person, :person_escort_record]
  Remove from your query: .includes([:person, :person_escort_record])
Call stack

2020-10-21 16:38:33[WARN] user: william
GET /api/moves/d17c348f-ae1c-4385-9899-0b711fe17e19?include=events
AVOID eager loading detected
  Person => [:gender, :ethnicity]
  Remove from your query: .includes([:gender, :ethnicity])
Call stack
```

**when indexing moves:**

```
GET /api/moves?include=events
AVOID eager loading detected
  Move => [:allocation, :supplier, :court_hearings, :prison_transfer_reason, :original_move, :person, :from_location, :to_location]
  Remove from your query: .includes([:allocation, :supplier, :court_hearings, :prison_transfer_reason, :original_move, :person, :from_location, :to_location])
Call stack

2020-10-21 16:40:28[WARN] user: william
GET /api/moves?include=events
AVOID eager loading detected
  Profile => [:documents, :person_escort_record]
  Remove from your query: .includes([:documents, :person_escort_record])
Call stack

2020-10-21 16:40:28[WARN] user: william
GET /api/moves?include=events
AVOID eager loading detected
  Document => [:file_attachment]
  Remove from your query: .includes([:file_attachment])
Call stack
```